es-space-mgmt/main/default/lwc/reservationTile/__tests__/reservationTile.test.js

```diff
@@ -0,0 +1,125 @@
+import { createElement } from 'lwc';
+import ReservationTile from 'c/reservationTile';
+
+const RESERVATION_CONTACT = require('./data/contactReservation.json');
+
+const RESERVATION_LEAD = require('./data/leadReservation.json');
+
+describe('c-reservation-tile', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('renders reservationTile for Lead', () => {
+        const element = createElement('c-reservation-tile', {
+            is: ReservationTile
+        });
+        element.reservation = RESERVATION_LEAD;
+        document.body.appendChild(element);
+
+        return Promise.resolve().then(() => {
+            // check for div class
+            const divElement = element.shadowRoot.querySelector('div.pointer');
+            expect(divElement.className).toBe('pointer');
```

Comment:
This isn't a good test given the fact that you selected the div with a `.pointer` selector.
Update the selector to just `div`.

Reply:
done!